language/proto/known_imports.go

```diff
@@ -7,16 +7,16 @@ import "github.com/bazelbuild/bazel-gazelle/label"
 
 var knownImports = map[string]label.Label{
 
-	"google/protobuf/any.proto":             label.Label{Repo: "com_google_protobuf", Pkg: "", Name: "any_proto"},
-	"google/protobuf/api.proto":             label.Label{Repo: "com_google_protobuf", Pkg: "", Name: "api_proto"},
-	"google/protobuf/compiler/plugin.proto": label.Label{Repo: "com_google_protobuf", Pkg: "", Name: "compiler_plugin_proto"},
-	"google/protobuf/descriptor.proto":      label.Label{Repo: "com_google_protobuf", Pkg: "", Name: "descriptor_proto"},
-	"google/protobuf/duration.proto":        label.Label{Repo: "com_google_protobuf", Pkg: "", Name: "duration_proto"},
-	"google/protobuf/empty.proto":           label.Label{Repo: "com_google_protobuf", Pkg: "", Name: "empty_proto"},
-	"google/protobuf/field_mask.proto":      label.Label{Repo: "com_google_protobuf", Pkg: "", Name: "field_mask_proto"},
-	"google/protobuf/source_context.proto":  label.Label{Repo: "com_google_protobuf", Pkg: "", Name: "source_context_proto"},
-	"google/protobuf/struct.proto":          label.Label{Repo: "com_google_protobuf", Pkg: "", Name: "struct_proto"},
-	"google/protobuf/timestamp.proto":       label.Label{Repo: "com_google_protobuf", Pkg: "", Name: "timestamp_proto"},
-	"google/protobuf/type.proto":            label.Label{Repo: "com_google_protobuf", Pkg: "", Name: "type_proto"},
-	"google/protobuf/wrappers.proto":        label.Label{Repo: "com_google_protobuf", Pkg: "", Name: "wrappers_proto"},
+	"google/protobuf/any.proto":             label.Label{Repo: "protobuf", Pkg: "", Name: "any_proto"},
```

Comment:
Is this change intentional? It looks like it could break the WORKSPACE setup - could you extract it out?

Reply:
I reverted this for now and the PR is green still, so I guess this isn't required in this PR (I originally had more changes in this PR and I guess didn't rollback everything such as this file).

This change will be required when running the update with go 1.22 which has a lot more changes then what's in this PR. That can be next...